classes/migration/upgrade/v3_4_0/I8866_DispatchRegionCodesFixingJobs.php

```diff
@@ -31,8 +37,9 @@ class I8866_DispatchRegionCodesFixingJobs extends Migration
      */
     public function up(): void
     {
-        if (DB::table('metrics_submission_geo_monthly')->whereNotNull('region')->exists() ||
-            DB::table('metrics_submission_geo_daily')->whereNotNull('region')->exists()) {
+        if (DB::table('metrics_submission_geo_monthly')->where('region', '<>', '')->exists() ||
+            DB::table('metrics_submission_geo_daily')->where('region', '<>', '')->exists()) {
+
```

Comment:
GitHub doesn't allow me to comment on the lines below, but adding an index to the temporary table's columns might be helpful to performance.